js/force-directed-graph.js

```diff
@@ -177,6 +177,9 @@ function generateForceDirected() {
     // d3.select(this).attr("r", 12);
 
     link.attr("opacity", function(d) {
+      // TODO I'm assuming we're setting opacity here. can we make the links opaque ONLY if the similarity is above the mean similarity in the whole dataset?
```

Comment:
@lilybhattacharjee5 I left a suggestion here for how we might want to do transparency on these edges. LMK what you think.